crates/evm/fuzz/src/strategies/calldata.rs

```diff
@@ -1,18 +1,65 @@
-use super::fuzz_param;
+use crate::strategies::{fuzz_param, EvmFuzzState};
 use alloy_dyn_abi::JsonAbiExt;
 use alloy_json_abi::Function;
-use alloy_primitives::Bytes;
+use alloy_primitives::{Address, Bytes};
+use foundry_config::FuzzDictionaryConfig;
+use hashbrown::HashSet;
 use proptest::prelude::{BoxedStrategy, Strategy};
+use std::{fmt, sync::Arc};
+
+pub type CalldataFuzzDictionary = Arc<CalldataFuzzDictionaryConfig>;
```

Comment:
I really disklike this alias style, this makes it much harder to navigate.

I know we have this in a few other places, but this isn't very helpful. I'd prefer a new wrapper type like

```rust
struct CalldataFuzzDictionary {
   inner: Arc<CalldataFuzzDictionaryConfig>
}
```

Reply:
done with https://github.com/foundry-rs/foundry/pull/7240/commits/b77dcb6fca13e732067cf50ac517e6c5fc641122 plus some code cleanup